docs/reference/edgectl-download.md

```diff
@@ -0,0 +1,42 @@
+# Downloads
+
+Edge Control is available for macOS, Linux, and Windows.
+
+Ensure that you have 
+
+## macOS
+
+1. To download by curl command, run the following in your command line:
+
+    ```shell
+    sudo curl -fL https://metriton.datawire.io/downloads/darwin/edgectl -o /usr/local/bin/edgectl && sudo chmod a+x /usr/local/bin/edgectl
+    ```
+
+2. Or, download the [binary file](https://metriton.datawire.io/downloads/darwin/edgectl).
+3. Move the file into your PATH with `sudo mv ./edgectl /usr/local/bin/edgectl`
+   * If you need to, print your PATH with `echo $PATH`
+4. Ensure the file is executable with the command `chmod a+x /usr/local/bin/edgectl`
+5. Run the executable with `./edgectl`
```

Comment:
You can't run ./edgectl if you downloaded edgectl to /usr/local/bin/edgectl.

Reply:
So where in their PATH should they move it? Is there a specific place we are recommending other than just "somewhere"?